### PR TITLE
[8.0] account_move_line_report_xls description update

### PR DESCRIPTION
**Pull request by luc-demeyer**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: 

Please check https://github.com/OCA/account-financial-reporting/pull/63 also for an updated icon.

Thanks

**Comment by yvaucher** [comment]
:+1:

File: account_move_line_report_xls/README.rst [hunk 1]
```diff
@@ -7,10 +7,10 @@ This module extends the functionality of the journal items
 Installation
```
Comment (lmignon) [inline account_move_line_report_xls/README.rst:10]: Would be nice to also have the licence badge on top of the description...

``` rest
.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
    :alt: License
```

**Comment by luc-demeyer** [comment]
I have updated the PR with the new icon as suggested by Pedro.
At the same time I also fixed the travis error in the account_financial_report_webkit module.

**Comment by tafaRU** [comment]
Would be nice also use  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst but the rest LGTM :+1:

**Comment by luc-demeyer** [comment]
@nbessi 
Travis fails because of broken yml test (aged_trial_balance.yml).
Can you have a look ?

**Comment by lmignon** [comment]
:+1:

**Comment by tafaRU** [comment]
@luc-demeyer please ignore my previous comment, diff have measled me :wink:

**Comment by nbessi** [comment]
@luc-demeyer hmm I'm not sure of the root cause. Maybe the default for period_from is not call.
It should take some time to debug.

**Comment by luc-demeyer** [comment]
@nbessi can this PR in the meanwhile be merged since travis error is independent from the PR ?

**Comment by nbessi** [comment]
@luc-demeyer I have no objection but it will report the problem later. At  least a bug report must be opened

**Comment by luc-demeyer** [comment]
@nbessi cf. https://github.com/OCA/account-financial-reporting/issues/67

**Comment by yvaucher** [comment]
Please rebase. Your PR shouldn't change icon as it was already done in #63

**Comment by luc-demeyer** [comment]
@yvaucher 
Rebase done. Thanks for remarking this.

**Comment by yvaucher** [comment]
@luc-demeyer Thanks for the change :+1: 

We can merge now

**Pull request merged**